Tensile/Source/lib/include/Tensile/ContractionProblem.hpp

```diff
@@ -139,12 +139,16 @@ namespace Tensile
                                                size_t   batchSize,
                                                size_t   lda,
                                                size_t   aStride,
+                                               size_t   aOffset,
```

Comment:
Hi @sdquiring,

Rocblas has use case to add offset to buffer address.

Why we need to add offset in API is because input buffer address in general batch gemm is array of buffer pointer which point to each batch_idx buffer. We have to add offset to each batch_idx buffer. If we don't add offset in Tensile Kernel, we have to allocate a temp GPU buffer, launch a kernel to add offset to temp buffer and pass to Tensile.
That's why I want to support offset in Tensile API.

We can use different construction to support origin API. Let me change it later.

Reply:
Yes, I'd recommend keeping these functions as-is, and adding new functions that include the offsets.

Reply:
ok, I got it.